Source/Core/VideoCommon/TextureCacheEntry.cpp

```diff
@@ -0,0 +1,124 @@
+#include "Common/Align.h"
```

Comment:
Copyright header

`<algorithm>` also needs to be included.

Reply:
This is done, thank you!